lib/action/model/subcorpus/__init__.py

```diff
@@ -113,22 +115,23 @@ async def create_subcorpus(self) -> Dict[str, Any]:
             worker = bgcalc.calc_backend_client(settings)
             res = await worker.send_task(
                 'create_subcorpus', object.__class__,
-                (self.session_get('user', 'id'), self.args.corpname, full_path, tt_query, imp_cql),
+                (author, self.args.corpname, full_path, tt_query, imp_cql, author, description),
                 time_limit=self.TASK_TIME_LIMIT)
             self.store_async_task(AsyncTaskStatus(
                 status=res.status, ident=res.id, category=AsyncTaskStatus.CATEGORY_SUBCORPUS,
                 label=f'{self.args.corpname}/{data.subcname}',
                 args=dict(subcname=data.subcname, corpname=self.args.corpname)))
-            result = {}
+            result = res.get()
```

Comment:
Please revert this one as we need fully asynchronous processing here. The possible additional bugs related to the processing will be resolved separately.